fluentgen/pom.xml

```diff
@@ -40,12 +40,6 @@
       <version>1.0.0-beta.1</version>
     </dependency>
 
-    <dependency>
-      <groupId>com.google.googlejavaformat</groupId>
-      <artifactId>google-java-format</artifactId>
-      <version>1.17.0-beta.1</version>
-    </dependency>
```

Comment:
fluent does not use postprocess nor Eclipse language server.

Maybe we first keep it here, and let @XiaofeiCao do the migration later.

I am a bit worried about the extra time when invoking Eclipse language server (it's pretty slow).

Reply:
Fortunately, Spotless doesn't require the Eclipse language server to perform Eclipse code formatting. Don't know the exact design but I know from investigating the removal of Google Format that Eclipse JDT Core has the ability to run code formatting without use of the language server and is just as fast as Google Format. Though, using Spotless requires code to be on disk rather than in memory, which will be slower but shouldn't be by much.

Based on that, thoughts on copying the code I added into postprocessor to fluent, and leaving the Google Format dependency out.

Reply:
Yeah, we can just copy the code to fluentgen.

Maybe another solution is to put the spotless related code to a module that can be used independently from postprocess (like current "partial-update" module). Anyway if we really want it we can always refactor later.

Reply:
I would save the larger refactor for later as there is other functionality copied across the plugins which would be useful to centralize.